src/couch_mrview_http.erl

```diff
@@ -229,7 +229,8 @@ do_all_docs_req(Req, Db, Keys, NS) ->
     end,
     Args = Args1#mrargs{preflight_fun=ETagFun},
     {ok, Resp} = couch_httpd:etag_maybe(Req, fun() ->
-        VAcc0 = #vacc{db=Db, req=Req},
+        Max = config:get_integer("httpd", "chunked_response_buffer", 1490),
```

Comment:
We don't have an official policy on it yet, but I've been trying to encourage people to wrap config calls behind functions, for example [1], especially given you're doing the same lookup three times here. The general idea is to minimize the number of places that are aware of the details of the config names, and also to avoid config name typos which have cropped up in the codebase (most recently seen with `shard_db` vs `shards_db`).


[1] https://github.com/cloudant/cassim/blob/master/src/cassim_metadata_cache.erl#L58-L59